Assets/_Project/Code/Models/BadgeList.cs

```diff
@@ -0,0 +1,30 @@
+﻿public class BadgeList
+{
+    [JsonProperty("badges")]
+    private List<Badge> badges;
+
+    public BadgeList()
+    {
+        badges = new List<Badge>();
+    }
+
+    public void AddBadge(Badge badge)
+    {
+        badges.Add(badge);
+    }
+
+    public Badge FindBadge(string name)

```

Comment:
warto oznaczyć to jako nullable "Badge?" żeby pokazać że zwracany moze być też null